test/sequential/test-stream2-fs.js

```diff
@@ -68,3 +68,16 @@ w.on('results', function(res) {
 });
 
 r.pipe(w);
+
+{
+  // Verify that end works when start is not specified.
```

Comment:
The test file here is named `test-stream2-fs` and it's in `sequential`. I think the test case added here belongs to `test/parallel/test-fs-read-stream.js` since it's more of a fs test, not a stream2 test. (Also I think this file can be moved to parallel? Although that should be done in another PR).

Reply:
ok. I just moved the new test case to `test/parallel/test-fs-read-stream.js`.